### ISSUE TITLE
Dates outside JS range breaks Studio

### ISSUE THREAD

**Issue by cmelchior** [realm/realm-studio#1094]
If your data model has a value that is apparently outside the dates supported by RealmJS, Studio fails to show anything from the selected table, but instead just shows "Invalid Time Value".

IMO, this should be restricted to just the single field and maybe rephrase to something like "Cannot show this value"

![image](https://user-images.githubusercontent.com/406066/52806719-04766980-308a-11e9-9365-0a9c84668534.png)

**Comment by nirinchev** [comment]
Or perhaps Studio/Realm js should represent dates as moment objects as those are likely capable of representing broader range.

**Comment by kraenhansen** [comment]
Can you upload a Realm containing an invalid date? Or perhaps provide an example on how to create such a value?

**Comment by cmelchior** [comment]
It happened after creating some random date values using `random.nextLong()` in Java... I suspect that using e.g. `Long.MAX_VALUE` will trigger it, but I'll try to create an example Realm.

**Comment by cmelchior** [comment]
[datetest.realm.zip](https://github.com/realm/realm-studio/files/2868931/datetest.realm.zip)

This Realm contains a class `DateTest` with 4 objects with the values:

```
Date(Long.MAX_VALUE)
Date(Long.MIN_VALUE)
Date(0)
Date.now()
```

It exhibits the same behaviour as above.
